test/e2e/common/pods.go

```diff
@@ -640,8 +655,13 @@ var _ = framework.KubeDescribe("Pods", func() {
 		}
 	})
 
+	/*
+	  Release : v1.12
+	  Testname: Pods: back-off capping.
+	  Description: Back-off must be capped at MaxContainerBackOff.
+	*/
 	// Slow issue #19027 (20 mins)
-	It("should cap back-off at MaxContainerBackOff [Slow][NodeConformance]", func() {
+	framework.ConformanceIt("should cap back-off at MaxContainerBackOff [Slow][NodeConformance]", func() {
```

Comment:
[This averages 27min per run](https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gke-slow&graph-metrics=test-duration-minutes&include-filter-by-regex=MaxContainerBackOff&width=5), which makes me cringe.